packages/seed/src/commands/clean/cleanOrphanedSeedFolders.ts

```diff
@@ -0,0 +1,171 @@
+        return new Set(fixtureConfig.map((config) => config.outputFolder));
+    }
+    return null;
+}
+
+function getActualSeedFolders(generator: GeneratorWorkspace): string[] {
+    const seedPath = generator.absolutePathToWorkspace;
+    const files = fs.readdirSync(seedPath);
+    return files.filter((file) => {
+        const fullPath = path.join(seedPath, file);
+        if (!fs.statSync(fullPath).isDirectory()) {
+            return false;
+        }
+        if (file === "node_modules" || file === ".git") {
+            return false;
+        }
+        return true;
+    });
+}
+
+function getSubFolders(folderPath: AbsoluteFilePath): string[] {
+    const files = fs.readdirSync(folderPath);
+    return files.filter((file) => {
+        const fullPath = path.join(folderPath, file);
+        return fs.statSync(fullPath).isDirectory();
+    });
+}
+
+export function findOrphanedSeedFolders(generators: GeneratorWorkspace[]): OrphanedFolder[] {
+    const testDefinitions = getTestDefinitions();
+    const orphanedFolders: OrphanedFolder[] = [];
+
+    for (const generator of generators) {
+        const expectedFixtures = getExpectedFixtures(generator, testDefinitions);
+        const actualFolders = getActualSeedFolders(generator);
+
+        for (const folder of actualFolders) {
+            const folderPath = join(generator.absolutePathToWorkspace, RelativeFilePath.of(folder));
+
+            if (!expectedFixtures.has(folder)) {
+                orphanedFolders.push({
+                    generator: generator.workspaceName,
+                    folder,
+                    absolutePath: folderPath
+                });
+                continue;
+            }
+
+            const expectedOutputFolders = getExpectedOutputFoldersForFixture(generator, folder);
+            if (expectedOutputFolders != null) {
+                const actualSubFolders = getSubFolders(folderPath);
+                for (const subFolder of actualSubFolders) {
+                    if (isLikelyOutputFolder(subFolder) && !expectedOutputFolders.has(subFolder)) {
+                        orphanedFolders.push({
+                            generator: generator.workspaceName,
+                            folder,
+                            subFolder,
+                            absolutePath: join(folderPath, RelativeFilePath.of(subFolder))
+                        });
+                    }
+                }
+            }
+        }
+    }
+
+    return orphanedFolders;
+}
+
+function isLikelyOutputFolder(folderName: string): boolean {
+    const sharedFolderPatterns = [
+        /^\./, // Hidden folders like .github, .mock
+        /^src$/, // Source folder
+        /^tests?$/, // Test folders
+        /^node_modules$/, // Node modules
+        /^__pycache__$/, // Python cache
+        /^\.venv$/, // Python virtual environment
+        /^dist$/, // Distribution folder
+        /^build$/, // Build folder
+        /^target$/ // Rust/Java target folder
```

Comment:
Are there actually any of these folders? I don't think they should exist.
It should be pretty deterministic.

If a test fixture exists, but no subfolder configuration seed.yml, it's just the fixture name.
If subfolders exists, the fixture folder and subfolder should exists.
You can easily build a full list of fixtures and subfolders like this, then any other folders should be deleted. Only _.gitignore_ and _seed.yml_ files should remain.

Reply:
It is intended to be cleaned up.
The reason this happened is because there used to not be a subfolder defined, then seed ran, then a subfolder was defined, and seed ran.
The result is that the non-subfolder seed generation was never cleaned up.
The point of this command is to clean exactly this up.